modules/hostap/src/supp_api.c

```diff
@@ -1139,6 +1149,72 @@ int supplicant_11k_neighbor_request(const struct device *dev, struct wifi_11k_pa
 	return 0;
 }
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_ROAMING
+int supplicant_spec_scan(const struct device *dev, struct wifi_scan_params *params)
+{
+	int i = 0;
+	char cmd[100] = {0};
+	char *pos = cmd;
+	char *end = pos + 100;
```

Comment:
Could you have a define for the array length, or alternatively use sizeof. It is error prone to hard code the value as they can become mismatched if changed later.
So we could say here `char *end = pos + sizeof(cmd);`

Reply:
Updated. Added new macro define for the array length.